### ISSUE TITLE
The English Repo Refugees Need Your Help

### ISSUE THREAD

**Issue by mpaunovic** [hexated/cloudstream-extensions-hexated#638]
Unfortunately The English Repo is no longer maintained. Have a look here:

https://codeberg.org/cloudstream/cloudstream-extensions/raw/branch/builds/repo.json

Could you please add support to your repo for sflix.to (and/or fmoviesz.to)? Perhaps any of the clones/mirrors?

[Sflix Clones](https://www.reddit.com/r/FREEMEDIAHECKYEAH/wiki/storage/#wiki_sflix_clones)

[Fmovies Clones](https://www.reddit.com/r/FREEMEDIAHECKYEAH/wiki/storage/#wiki_fmovies_clones)

We would be very grateful if you could help us.

Thank you!

**Comment by erikk333** [comment]
Most of the sources available at fmoviesz and sflix are accesiible through sorastream. I tink vidcloud is avaialble in sorastream for example
![afbeelding](https://github.com/hexated/cloudstream-extensions-hexated/assets/147526653/d09f030b-94e5-4c4a-beb6-1fe562efafaf)

**Comment by phisher98** [comment]
Yup Sflix doesn't even worked in Eng Repo and Sora is Best Extension in CS try that if solved you can close this issue then @mpaunovic
